pkg/antctl/raw/proxy/command_test.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package proxy
+
+import (
+	"bytes"
+	"fmt"
+	"io/fs"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/klog/v2"
+)
+
+func TestValidateAndComplete(t *testing.T) {
+	tempfile, err := os.CreateTemp("", "tmpfile")
+	if err != nil {
+		t.Errorf("error creating tempfile: %v", err)
+	}
+
+	fileinfo, err := tempfile.Stat()
```

Comment:
Rename tempfile to tempFile and fileinfo to fileInfo.

Reply:
done, thanks!